misc/graylog.conf

```diff
@@ -746,3 +746,17 @@ mongodb_max_connections = 1000
 # Maximum value that can be set for an event limit.
 # Default: 1000
 #event_definition_max_event_limit = 1000
+
+# Optional limits on scheduling concurrency by job type. No more than the specified number of worker
+# threads will be executing jobs of the specified type across the entire cluster.
+# Default: no limitation
+# Note: Monitor job queue metrics to avoid excessive backlog of unprocessed jobs when using this setting!
+# Available job types:
```

Comment:
If we advertise the feature here, it should also work in Graylog open, so the code for parsing it needs to be moved to graylog2-server.

We should not list job types from the enterprise plugin here, those won't be available when the enterprise plugin is not present. Enterprise specific job types should probably be listed in our documentation then.

Reply:
Done - apart from the docs update.